### ISSUE TITLE
Regarding the MIT License of the models

### ISSUE THREAD

**Issue by ssrp** [kpzhang93/MTCNN_face_detection_alignment#55]
Hi, thank you for this work and the source code!

Regarding the MIT license of this repo -- if you have used FDDB/WIDERFace dataset for training your models (which are provided only for the research purposes, not commercial usage), how can the weights you provide be on the MIT license?

MIT license allows anyone to use this commercially, which is a derivative work of the images which are not allowed to be used commercially.

Looking forward to your reply. Thanks!

**Comment by ssrp** [comment]
Hi, a gentle reminder to my request. I would really like to have clarification on this.

**Comment by lynnwilliam** [comment]
I agree with you, if the source images for training can't be used for commercial use.
Any work derived from those images can't be used for commercial work either.
I don't expect you will get a reply.
